### ISSUE TITLE
Missing Ribbon menu after updating

### ISSUE THREAD

**Issue by Ruuno** [joyfullservice/msaccess-vcs-addin#477]
I updated my Add-In version from v4.0.25 to the newest v4.0.30. Then i wanted to try the new Split Layout from VBA function which, when first applied, shows this message:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/95e31ab4-a863-40fa-a82a-74114321a3d0)

After i went to the linked wiki article i tried to follow the steps but noticed that i dont have the split tool in my advanced tools section in the ribbon:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/13cf982b-fe54-4d35-ab34-111f83273982)

Now i was confused and wondered if the Add-In even works as expected. I unchecked the split option again and tried source exporting. It worked.
Then i tried to troubleshoot by full uninstalling  then installing v4.0.30 while keeping my settings.
The uninstall went fine. Upon starting the new install of v4.0.30 and as i did before with v4.0.25 i checked both advanced options:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/73e7cdf7-d71a-4b1b-87d0-165435bf89f6)

then clkcking on Ok when asked to add the trusted location i am met with this:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/98af570e-8ecc-4aef-94ca-a7289b5b57d9)

After pressing OK i still get the successful install message as such:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/71cdd8b2-c334-47cd-9093-3ac2d7f840bf)

But when i navigate to the install location i noticed this issue:

![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/ef834901-30d2-4101-a5ef-8f1077e13e72)

There is no Ribbon.xml or dll file present. I assume when going through the update path while having v4.025 installed the ribbon and dll also did not get updated and thats why i could not find the split tool in the ribbon.
Now i could not even uninstall the addin via the ribbon because it is missing. And when i try to open it to uninstall via the Add-Ins Dropdown i get this:
![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/ed781e12-df2e-4830-a9b1-f61334bc6e96)
![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/7f6a4699-af7f-426f-a164-89a356126c51)
![grafik](https://github.com/joyfullservice/msaccess-vcs-addin/assets/123254223/535e4571-3e00-4ee2-b406-25287a8023f4)

So my last resort is to try and uninstall the tool another way to atleast get my old v4.0.25 working.

I am sorry for the lengthy post but i wanted to make sure to be as detailed as possible. I can provide more info if needed. I tried debugging but could not find the point of error.

**Comment by joyfullservice** [comment]
Thank you for the detailed notes on this issue! I will plan to look into this later today...

**Comment by joyfullservice** [comment]
Thanks again for reporting this! It looks like what happened is that the binary that I attached to this release was missing the entries in the resources table. I have updated the binary after verifying that these records exist.

![image](https://github.com/joyfullservice/msaccess-vcs-addin/assets/7254992/c6d77a90-de08-401f-9cef-adccd35674fc)

My guess is that some of the other errors may be related to the missing resource files. Let me know if it installs and works normally after downloading a fresh copy of the binary from the releases page.

Also, (just so you know), in most environments you won't need that second checkbox option in Advanced Options to open the add-in file after installing. That was added as a workaround for some specific user environments. (Leaving this unchecked will save you a step when installing.)
